### PR TITLE
Feature/summary page design

### PR DESCRIPTION
**Pull request by MohamedWael8**

### CODE REVIEW

**Comment by MohamedWael8** [comment]
Wait for checks to resolve first before merging

File: package.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
   "private": true,
   "dependencies": {
     "@babel/core": "7.4.5",
-    "@fortawesome/fontawesome-svg-core": "1.2.19",
-    "@fortawesome/free-solid-svg-icons": "5.9.0",
-    "@fortawesome/react-fontawesome": "0.1.4",
+    "@fortawesome/fontawesome-svg-core": "^1.2.19",
```
Comment (msafwat) [inline package.json]: Can you please move the ^

File: src/components/Timeline/Timeline.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+              {prop.title ? (
+                <div className={classes.timelineHeading}>
+                  <Badge color={prop.titleColor}>{prop.title}</Badge>
+                </div>
+              ) : null}
+              <div className={classes.timelineBody}>{prop.body}</div>
+              {prop.footerTitle ? (
+                <h6 className={classes.footerTitle}>{prop.footerTitle}</h6>
+              ) : null}
+              {prop.footer ? <hr className={classes.footerLine} /> : null}
```
Comment (msafwat) [inline src/components/Timeline/Timeline.jsx]: Can be combined with next line

File: src/redux/actions/summaryActions.test.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ var axiosMock = new MockAdapter(axios);
 describe("Async Actions", () => {
   beforeAll(() => {
     process.env.REACT_APP_API_GATEWAY =
-      "https://pi9joivue7.execute-api.us-east-1.amazonaws.com/";
+      "https://pi9joivue7.execute-api.us-east-1.amazonaws.com/test/";
```
Comment (msafwat) [inline src/redux/actions/summaryActions.test.js]: Should we use test or test-test?

File: src/views/Sustainability/NewAssessmentPage.jsx [hunk 4]
```diff
@@ -45,9 +45,28 @@ const NewAssessmentPage = ({
             updatedProps.assessmentId = uuidv1();
             return updatedProps;
           }
-          case "setSummariesList": {
+          // case "setSummariesList": {
```
Comment (msafwat) [inline src/views/Sustainability/NewAssessmentPage.jsx]: Do you need this comment anymore

File: src/views/Sustainability/WizardSteps/Summary.jsx [hunk 5]
```diff
@@ -40,7 +53,125 @@ const style = {
...
+  //assessment.cropZones.forEach(cropZone => {});
+  const widgetStories = [
+    {
+      inverted: true,
+      badgeColor: "danger",
+      badgeIcon: Create,
+      title: "Basic Info",
+      titleColor: "info",
+      body: (
+        // Design One Simplistic
```
Comment (msafwat) [inline src/views/Sustainability/WizardSteps/Summary.jsx]: needed?

File: src/redux/actions/summaryActions.test.js (hunk 3 above)
Comment (MohamedWael8) [reply src/redux/actions/summaryActions.test.js]: this is a mock adapter for calling the mock it doesn't actually call the api, and we are currently working with the test api gateway so even if it was test would be correct

File: src/views/Sustainability/NewAssessmentPage.jsx (hunk 4 above)
Comment (MohamedWael8) [reply src/views/Sustainability/NewAssessmentPage.jsx]: yes if we need to revert back to the old design we will need this

File: src/views/Sustainability/WizardSteps/Summary.jsx (hunk 5 above)
Comment (MohamedWael8) [reply src/views/Sustainability/WizardSteps/Summary.jsx]: until mohamed reda decides on a certain style we will leave it

File: package.json (hunk 1 above)
Comment (MohamedWael8) [reply package.json]: you mean remove ?

File: src/components/Timeline/Timeline.jsx (hunk 2 above)
Comment (MohamedWael8) [reply src/components/Timeline/Timeline.jsx]: this is the timeline component from the dashboard it is not a custom component that we created

File: package.json (hunk 1 above)
Comment (msafwat) [reply package.json]: yeah

File: src/redux/actions/summaryActions.test.js (hunk 3 above)
Comment (msafwat) [reply src/redux/actions/summaryActions.test.js]: ok np, if you certain that. later anyway we will move test-test to test

**Pull request merged**